artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ArtistQuery do
+  def main(args) do
+    start_query(args)
+  end
+
+  defp start_query(music_file) do
+    artist = String.upcase(prompt)
+    load_songs(music_file)
+    |> Enum.filter_map(&(String.upcase(&1["Artist"])) == artist, &(&1["Name"]))
+    |> print_results(titlecase(artist))
+
+    start_query(music_file) # Repeat forever
```

Comment:
Thoughts on calling this `query_loop` or something like that? Then the name is more indicative of what it's doing from the calling function, and you don't need this comment.